### DISCUSSION TITLE
Problem with custom ranges with images

### DISCUSSION THREAD

**Discussion by Samu1055** [Kas-tle/java2bedrock.sh#3627] [Converter Help] [answered]
Hello, I have tried to convert my java texture pack to bedrock in every possible way to have ranges in the tab with images, in java it works perfectly but in bedrock I don't know if there is support with the image unicodes

![image](https://user-images.githubusercontent.com/107415822/215243222-88cfb626-7e59-4e1d-a366-e16a32037541.png)
![image](https://user-images.githubusercontent.com/107415822/215243244-56ef5d26-ee10-47da-9c3e-9fe5291a0c86.png)
![image](https://user-images.githubusercontent.com/107415822/215243311-f077744e-7d17-4c7d-9f73-d9ad0ffb6cb2.png)

**Comment by Kas-tle** [comment, marked as answer]
Per the Readme, this converter only handles models defined with damage or custom model data predicates. For custom emoji, please refer to the Bedrock Wiki https://wiki.bedrock.dev/concepts/emojis.html.

**Reply by sdawd2q3e** [reply]
This is an incomplete answer. He should explain the size he needs from the glyph and how he can make the glyph

> في المستند التمهيدي ، يتعامل هذا المحول فقط مع النماذج المحددة بالضرر أو مسندات بيانات النموذج المخصص. للحصول على رموز تعبيرية مخصصة ، يرجى الرجوع إلى Bedrock Wiki https://wiki.bedrock.dev/concepts/emojis.html .

This is an incomplete answer. He should explain the size he needs from the glyph and how he can make the glyph

**Reply by Kas-tle** [reply]
The bedrock wiki explains this. Please refer to the link provided.
